src/MobileHeader.jsx

```diff
@@ -41,6 +42,7 @@ class MobileHeader extends React.Component {
             key={`${type}-${content}`}
             className={`nav-link${disabled ? ' disabled' : ''}${isActive ? ' active' : ''}`}
             href={href}
+            onClick={onClick || null}
```

Comment:
would `null` cause any issue or any console error/warning?

Reply:
its not causing any issue locally but if you think we should do something else please suggest me

Reply:
if there is no functionality issue and no console error/warning then this is good.

Reply:
I'd like to understand why the explicit `|| null` is necessary, if you folks don't mind.  It seems it can't be `undefined`, which would likely make things fail.  Can it ever be `''` or `0`, which looks like what you're protecting against?